src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -203,10 +220,18 @@
              :receiver                  (address/get-shortened-key to-address)}])
          [quo/bottom-actions
           {:actions          :one-action
-           :button-one-label button-one-label
+           :button-one-label (if no-routes-found?
+                               (i18n/label :t/try-again)
+                               button-one-label)
            :button-one-props (merge button-one-props
-                                    {:disabled? confirm-disabled?
-                                     :on-press  on-confirm})}]
+                                    {:disabled? (and (not no-routes-found?) confirm-disabled?)
+                                     :on-press  (if no-routes-found?
+                                                  #(rf/dispatch [:wallet/get-suggested-routes
+                                                                 {:amount (controlled-input/input-value
+                                                                           input-state)}])
+                                                  on-confirm)}
```

Comment:
If no routes where found (because of an error or because the getSuggestedRoutes returned a success response with empty routes), we show the user a `Try again` button so they can resend the request

Reply:
Is this something from the designs?

Reply:
Yes, it is in the "No routes found" design